sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -262,6 +272,10 @@ case class Cos(child: Expression) extends UnaryMathExpression(math.cos, "COS")
 
 @ExpressionDescription(
   usage = "_FUNC_(expr) - Returns the hyperbolic cosine of `expr`.",
+  arguments = """
+    Arguments:
+      * expr - hyperbolic angle.
```

Comment:
not a big deal but let's remove the trailing `.` for consistency.

Reply:
ok